lib/iex/lib/iex/introspection.ex

```diff
@@ -141,6 +146,18 @@ defmodule IEx.Introspection do
     end
   end
 
+  defp callback_module(mod, fun, arity) do
+    mod.module_info(:attributes)
+    |> Keyword.get_values(:behaviour)
+    |> Stream.concat()
+    |> Stream.filter(fn(module)->
+      callbacks = module.module_info(:attributes)
```

Comment:
Do you mean:

```
callbacks = Enum.filter_map(module.module_info(:attributes), &match?({:callback, _}, &1), fn {_, [{t,_}|_]} -> t end)
```
Not sure if its any more readable since its kinda mashed together now... 

I could indent the next line, would that help?